blip-project/src/components/Page/Modal/ModalJoin.jsx

```
@@ -0,0 +1,165 @@
+      window.removeEventListener("keydown", handleKeyDown);
+    };
+  }, [onClose]);
+
+  useEffect(() => {
+    try {
+      const storedTeams = JSON.parse(localStorage.getItem("teams") || "[]");
+      if (storedTeams.length > 0) {
+        const latestTeam = storedTeams[storedTeams.length - 1];
+        if (latestTeam && latestTeam.JoinUrl) {
+          console.log("로컬 스토리지에서 JoinUrl 발견:", latestTeam.JoinUrl);
+          setJoinUrl(latestTeam.JoinUrl);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)